experian-fraud-stub/src/main/java/uk/gov/di/ipv/stub/fraud/InMemoryDataStore.java

```diff
@@ -19,8 +19,8 @@ public class InMemoryDataStore {
     private static final Logger LOGGER = LoggerFactory.getLogger(InMemoryDataStore.class);
     private final HashMap<String, IdentityVerificationResponse> experianResponses = new HashMap<>();
 
-    public InMemoryDataStore(ObjectMapper mapper) {
-        init();
+    public InMemoryDataStore(ObjectMapper mapper, boolean forPep) {
```

Comment:
Rather than a bool could we pass a string so this is a bit more extendable

Reply:
Changed to predefined strings - used here and in the check lookup.